webcommon/libs.graaljs/external/icu4j-24.0.0-license.txt

```diff
@@ -2,8 +2,8 @@ Name: icu4j
 Description: icu4j license
 License: MIT-icu4j-58
 Origin: http://site.icu-project.org/
-Version: 67.1
-Files: icu4j-67.1.jar
+Version: 24.0.0
+Files: icu4j-24.0.0.jar
 
```

Comment:
The license here does not match the license in the binary, in a significant way, I think. My suggestion here would be to use the text from here:
https://raw.githubusercontent.com/unicode-org/icu/248b1c2a13fc7fc346e9e9419c4794df8a590ad9/LICENSE
and remove everything related to ICU4C, i.e. everything from line `File: aclocal.m4 (only for ICU4C)`. That should mostly match the LICENSE, and hopefully match what we use.

Reply:
The license wording has changed for 74 release and above. I've added the license text  in 46848b4b58